src/jit/lsra.cpp

```diff
@@ -3689,7 +3694,6 @@ void LinearScan::buildRefPositionsForNode(GenTree*                  tree,
     {
         lsraDispNode(tree, LSRA_DUMP_REFPOS, (produce != 0));
```

Comment:
I responded to this in Codeflow but it seems to have gotten lost:
Wow, somehow I did a really bad job with the merge. Those were gone, but came back. I've reworked it just a bit and will post before and after dump excerpts.